.github/dependabot.yml

```diff
@@ -10,7 +10,7 @@ updates:
         update-types: ['version-update:semver-major']
     open-pull-requests-limit: 3
     commit-message:
-      prefix: "deps(npm): DSW-000 update"
+      prefix: "deps(npm): DSW-000"
```

Comment:
before it was clear it was a dependency update, how come you want to remove the work?

Reply:
It looks like dependabot naming is like so: `deps(github-actions): DSW-000 update: Bump slackapi/slack-github-action from 1.27.0 to 1.27.1 in /.github/workflows`, so the `update` isn't needed IMO